drive/tests/query_tests.rs

```diff
@@ -491,8 +514,8 @@ fn test_family_basic_queries() {
         .expect("there is always a root hash");
 
     let expected_app_hash = vec![
-        232, 7, 232, 41, 119, 242, 13, 28, 68, 33, 95, 195, 64, 85, 5, 185, 117, 93, 167, 217, 17,
-        11, 250, 239, 39, 21, 216, 146, 123, 5, 57, 151,
+        25, 64, 187, 169, 51, 20, 193, 193, 55, 86, 125, 179, 23, 202, 238, 71, 117, 68, 0, 29,
+        126, 98, 10, 240, 119, 142, 230, 78, 106, 38, 0, 0,
```

Comment:
All hashes are going to change now because you've changed the root tree structure from the default.

Reply:
Yeah, it will be last time since it's only depends on contract tree now.